CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to Headset
+
+Welcome to the Headset community!
+
+## Community
+
+We have two main channels of communication:
+- our Slack workspace: https://tinyurl.com/y7m8y5x4
+- issues on GitHub
+
+If you have any questions on where to start and/or how to proceed, let us know!
+
+## Ways to contribute
+
+We welcome many ways of contributing to Headset:
+- finding bugs and [reporting issues](https://github.com/headsetapp/headset-electron/issues)
+- picking and working on an existing [issue](https://github.com/headsetapp/headset-electron/issues) (the ones tagged
+  "help wanted" are usually a good place to start!)
+- proposing and implementing improvements to Headset
+
+## Setting up the environment
+
+1. Fork the repository on GitHub
+
+2. Clone your fork of the repository
+
+  ```
+  git clone https://github.com/<your-username>/headset-electron.git
+  ```
+
+3. Installation is OS-dependent, so choose the appropriate OS directory:
+  - `windows` for Windows
+  - `darwin` for OSX
+  - `linux` for Linux
+
+    and move to it with `cd <os-directory>`.
+
+4. Install dependencies:
+
+  ```
+  npm install
```

Comment:
It's recommended that you also do `npm install` on the root folder because the tests and `eslint` are run from root. Any contribution will be checked by the CI and running the test locally will speed things up

Reply:
It makes sense to install all the required dependencies -- it seems that the process is getting a tad complicated, especially for Linux, since it requires 3 separate steps. Would it be sensible to add a single installation script, that detects the OS and acts accordingly? The easier it is to set up the environment, the greater the chance someone will contribute.

Reply:
We use to have some scripts but we deleted them because we needed to run steps in-between the commands. Also, some people might have some of the modules installed globally, this is especially true for `eslint`.
Instead of mentioning a simple `npm install`, you can say to run this from the root:
```
npm i && cd <os> && npm i
```
where `<os>` is either `darwin`, `linux` or `windows`